src/middlewares/wallet.middleware.js

```diff
@@ -51,6 +51,17 @@ async function walletMiddleware(req, res, next) {
   // Adding to req parameter, so we don't need to get it in all requests
   req.wallet = wallet;
   req.walletId = walletId;
+
+  /* istanbul ignore if  */
```

Comment:
Why is this needed?

Reply:
It was meant for the if below, since it's unreachable in normal circunstances.
This was just to remove some lines from coverage that could not be tested.